### DISCUSSION TITLE
Quarkus at DevConf.CZ - Virtual event - January 28-29, 2022

### DISCUSSION THREAD

**Discussion by insectengine** [quarkusio/quarkus#23220] [Events]
[DevConf.CZ 2022](https://www.devconf.info/cz/) is the 14th annual, free, Red Hat sponsored virtual community conference for developers, admins, DevOps engineers, testers, documentation writers and other contributors to open source technologies. At DevConf.CZ, FLOSS communities sync, share, and hack on upstream projects together.

There is no admission or ticket charge for DevConf.CZ events. However, you are required to complete a free registration. 

We are committed to fostering an open and welcoming environment at our conference. We set expectations for inclusive behavior through our code of conduct and media policies, and are prepared to enforce these.

**[Register Today!](https://hopin.com/events/devconf-cz-2022)**

****Quarkus Speakers and Topics:****

_**Friday, January 28th**_

**[Quarkus Super-Heroes Workshop](https://devconfcz2022.sched.com/event/siHj/quarkus-super-heroes-workshop)**
Speaker: Martin Štefanko
Time: 9:30am CET

**[Enter Serverless Functions Journey with Quarkus](https://devconfcz2022.sched.com/event/siFk/enter-serverless-functions-journey-with-quarkus)**
Speaker: Daniel Oh
Time: 3:00pm CET

**[OptaPlanner AI on Quarkus for school timetabling
](https://devconfcz2022.sched.com/event/siGE/optaplanner-ai-on-quarkus-for-school-timetabling)** Speaker: Geoffrey De Smet Time: 4:30pm CET

**[Coding in Java with Joy - A Quarkus talk](https://devconfcz2022.sched.com/event/siJ2/coding-in-java-with-joy-a-quarkus-talk)**
Speaker: Shaaf Syed
Time: 5:00pm CET

_**Saturday, January 29th**_

**[Serverless Event Driven and Cloud Native Java
](https://devconfcz2022.sched.com/event/siEv/serverless-event-driven-and-cloud-native-java)** Speaker: Ali Ok
Time: 10:00am CET

**Comment by GavinRay97** [comment]
Will these talks be available on YouTube afterwards? I just noticed this announcement today and some of these I would have been interested in attending.

I assume so but wanted to double-check.

Thank you =)

**Reply by maxandersen** [reply]
I think most of them are available on devconf youtube channel by now https://www.youtube.com/c/DevConf_INFO/videos

i.e. I found https://www.youtube.com/watch?v=WzO93lPJXhw

**Reply by insectengine** [reply]
Sorry for the delayed response.

DevConf.CZ 2022 sessions can be found on Youtube:
https://www.youtube.com/playlist?list=PLU1vS0speL2bf9Zfa0pHQLPBhEKbbbsua
